src/platform/x11.rs

```diff
@@ -260,6 +260,7 @@ impl WindowBuilderExtX11 for WindowBuilder {
     }
 
     #[inline]
+    #[cfg(feature = "x11")]
```

Comment:
Nit: Remove

Reply:
[refactor: remove unnecessary cfg](https://github.com/rust-windowing/winit/pull/2441/commits/f565a87b7c89505f4f907be3ec4bf314b641f109)
I fixed this here